op-service/txmgr/kms.go

```diff
@@ -25,20 +26,36 @@ type KmsConfig struct {
 }
 
 func NewKmsConfig(cfg CLIConfig) (*KmsConfig, error) {
-	if cfg.KmsKeyID == "" || cfg.KmsEndpoint == "" || cfg.KmsRegion == "" {
-		return nil, fmt.Errorf("KMS config is not set")
+	var (
+		sess *session.Session
+		err  error
+	)
+	// AWS uses IAM role for task
+	if cfg.KmsProduction {
+		log.Info("Using AWS KMS production mode")
+		if cfg.KmsKeyID == "" {
```

Comment:
Since we are still referencing the region, shouldn't that be checked here to be non-empty?

Reply:
Yes, it has already been checked.
https://github.com/bobanetwork/v3-anchorage/blob/b186efbb4d3848185316a12ea73299fcbb5b91f5/op-service/txmgr/cli.go#L254-L261

Reply:
Yes, I added another check here.
https://github.com/bobanetwork/v3-anchorage/blob/3be4cda8c489fd24cdd7599b302a1dd3b08baa6d/op-service/txmgr/kms.go#L34-L41